package.json

```diff
@@ -4,27 +4,52 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start-dev": "nodemon ./server/app.js --exec babel-node",
+    "eslint": "eslint ./"
```

Comment:
We can use eslint and prettier to manage formatting instead of manually fixing syntax errors.